xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/uicomponents/extension/extension.css

```diff
@@ -475,3 +471,10 @@ ul.collapsible.document-tree {
   font-weight: normal;
   padding-right: .3em;
 }
+
+@media (min-width: 992px) {
```

Comment:
You should mention in a comment the Bootstrap LESS variable that corresponds to this value (otherwise it looks too much like a magic number). I'm also wondering whether it's not better to move this to the skin (Flamingo) in order to be able to use LESS (introducing a WebJar for the Extension Manager is more work).

Reply:
1c504fe adds a comment indeed. We could move the CSS rules to the skin indeed, should that be in `resources/flamingo/admin.less`? The commit also applies the same layout change to the extension dependency panel, I'm adding a screenshot to the related JIRA issue itself.

Reply:
A comment is fine for now I think, taking into account that this value is rarely changed from the color theme (close to never).